[2021-08-12 15:36:10,961] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:28:32.935438+00:00 [queued]>
[2021-08-12 15:36:10,986] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:28:32.935438+00:00 [queued]>
[2021-08-12 15:36:10,986] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:36:10,986] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-12 15:36:10,986] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 15:36:11,007] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_5> on 2021-08-12T13:28:32.935438+00:00
[2021-08-12 15:36:11,010] {standard_task_runner.py:52} INFO - Started process 11959 to run task
[2021-08-12 15:36:11,031] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag_03', 'task_5', '2021-08-12T13:28:32.935438+00:00', '--job-id', '144', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/task_example.py', '--cfg-path', '/tmp/tmpy4j0nusp', '--error-file', '/tmp/tmpwuui6bym']
[2021-08-12 15:36:11,032] {standard_task_runner.py:77} INFO - Job 144: Subtask task_5
[2021-08-12 15:36:11,107] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag_03.task_5 2021-08-12T13:28:32.935438+00:00 [running]> on host adam
[2021-08-12 15:36:11,193] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag_03
AIRFLOW_CTX_TASK_ID=task_5
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:28:32.935438+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:28:32.935438+00:00
[2021-08-12 15:36:11,194] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-12 15:36:11,195] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo hello from airflow > ~/Documents/hello_airflow.txt']
[2021-08-12 15:36:11,209] {subprocess.py:74} INFO - Output:
[2021-08-12 15:36:11,214] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-12 15:36:11,246] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=parallel_dag_03, task_id=task_5, execution_date=20210812T132832, start_date=20210812T133610, end_date=20210812T133611
[2021-08-12 15:36:11,278] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-12 15:36:11,314] {local_task_job.py:149} INFO - Task exited with return code 0
